### PR TITLE
Update to asciidoctor.js 1.5.2 and opal 0.6.3

### PR DESCRIPTION
**Pull request by jmini**
See [issue #22](https://github.com/asciidoctor/asciidoctor-firefox-addon/issues/22)

### CODE REVIEW

File: data/asciidoctor.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* Asciidoctor default stylesheet | MIT License | http://asciidoctor.org */
-@import "https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic|Noto+Serif:400,400italic,700,700italic|Droid+Sans+Mono:400";
+/* Remove the comments around the @import statement below when using this as a custom stylesheet */
+/*@import "https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400";*/
```
Comment (mojavelinux) [inline data/asciidoctor.css]: This `@import` line should be uncommented when used in the Firefox add-on. The reason is the Firefox addon renders the document to embeddable HTML, which does not include the fonts. This is kind of a pain atm, but suffice to say, it's a necessary manual change.

**Comment by mojavelinux** [comment]
Uncomment the `@import` line for the fonts and this will be good to go!

**Comment by Mogztter** [comment]
Thanks again @jmini :+1:

**Comment by jmini** [comment]
If I understand the GitHub workflow correctly, I have forced pushed an amended commit on my master branch. This new commit contains the fix asked by @mojavelinux

**Comment by mojavelinux** [comment]
You did it exactly right! Good job.

**Comment by Mogztter** [comment]
Perfect :beers:

**Pull request merged**